s3fs/core.py

```diff
@@ -1433,38 +1435,63 @@ def flush(self, force=False, retries=10):
             if force:
                 self.forced = True
 
-            self.buffer.seek(0)
-            part = len(self.parts) + 1
-            i = 0
-
             try:
                 self.mpu = self.mpu or self._call_s3(
                     self.s3.s3.create_multipart_upload,
                     Bucket=self.bucket, Key=self.key, ACL=self.acl)
-            except (ClientError, ParamValidationError) as e:
-                raise_from(IOError('Initiating write failed: %s' %
-                                   self.path), e)
+            except (ClientError, ParamValidationError) as exc:
+                raise_from(IOError('Initiating write failed: %s' % self.path), exc)
```

Comment:
Why the reformatting? I notice this in a number of places, perhaps it was automatically done by your IDE?

Reply:
Yep, I was uncertain what to do about that stuff – the `vim-flake8` plugin, (as configured of course), complained about a couple things: one-letter variables and unused variables. (And then I'm sure other settings resulted in different formatting.)

But, certainly, I'm happy to narrow the scope of changes, to exclude such things, in so far as they're unrelated to support for large writes.